cards/src/main/resources/cards/custom/Musician/hero_power_chord.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Chord",
+  "heroPower": "hero_power_polychord",
+  "baseManaCost": 2,
+  "type": "HERO_POWER",
+  "heroClass": "CORAL",
+  "rarity": "FREE",
+  "targetSelection": "NONE",
+  "description": "Give a minion +1/+1. If you have none, summon a 1/1 Patron.",

```

Comment:
i'll add the appropriate aura to this (auras also need to be written as being in play when written on a hero power)